staging/src/github.com/kubeedge/mapper-framework/_template/mapper/device/device.go

```diff
@@ -29,6 +29,7 @@ type DevPanel struct {
 	deviceMuxs   map[string]context.CancelFunc
 	devices      map[string]*driver.CustomizedDev
 	models       map[string]common.DeviceModel
+	protocols    map[string]common.ProtocolConfig
```

Comment:
In #5305 we have removed the protocols field. It seems you made some mistakes while rebasing @cl2017